hooks/post_gen_project.py

```diff
@@ -25,6 +25,14 @@
 DOCUMENTATION_ENGINE = "jupyter-book"
 {% endif %}
 
+
```

Comment:
you probably should do it in different way .. for example:

```python
{% if cookiecutter.use_bandit not in ["yes", "y"] -%}
USE_BANDIT = True
{% else %}
USE_BANDIT = False
{% endif %}
```

and inside the function:

```python
def bandit_clean_up():
    if not USE_BANDIT:
        return
    CS_PATH = PROJECT_DIRECTORY / '.bandit'
    remove_file(CS_PATH)
```

PS: remove the "if" inside the post_gen function https://github.com/osl-incubator/cookiecutter-python/pull/87/files#diff-8601710441b0dde909812d3f5a1d83245fec1f1bf94b6e510d1b72cbd2fb86c1R117